imperial_coldfront_plugin/views.py

```diff
@@ -585,6 +586,15 @@ def add_rdf_storage_allocation(request):
                 value=project_id,
             )
 
+            dart_id_attribute_type = AllocationAttributeType.objects.get(
+                name="DART ID", is_changeable=False
```

Comment:
Question: Did we want to have the `is_private=True` set here? Would imply that the end users won't be to see the attribute?

```
If you do not want the PI and allocation users to see a particular attribute, check the "Is private" checkbox. System administrators will be able to view these attributes on the allocation detail page but they'll be marked with a lock icon. Any others viewing the allocation will not see them at all.
```

Reply:
No, shouldn't be private.